fineract-investor/src/main/resources/db/changelog/tenant/module/investor/parts/0010_external_transafer_status_external_transfer_id_constraints.xml

```diff
@@ -22,7 +22,7 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
-    <changeSet author="fineract" id="1" context="mysql">
+    <changeSet author="fineract" id="1">
```

Comment:
this is an existing changeset that's been already executed on mysql.
In this case, mysql won't touch it, and PostgreSQL will recognize it as a new changset and just execute it without any problem?

Reply:
Originally there was no context, but at a later stage by mistake the context=mysql was added.
Now i remove the context again to avoid conflicts and checksum issues.

Reply:
got it, thanks for the clarification